docs/basics/101-136-cheatsheet.rst

```diff
@@ -0,0 +1,187 @@
+.. _cheat:
+
+The DataLad Cheat-Sheet
+-----------------------
+
+DataLad is a data management and publication multitool based
+on Git and Git-annex with a command line interface and a Python
+API. With DataLad, you can version control arbitrarily large data,
+share or consume data, and record changes made to data.
+
+**Basic command structure**:
+
+``datalad [-c <config-option>] COMMAND [--flag <optional flag specification>] [PATH]``
```

Comment:
IMHO useful to accent that there are options to datalad itself (-c -f -l) for any datalad invocation and command specific options. And those are separate and must not be confused (eg -f) since could have different meaning depending where specified (for datalad or command)